### ISSUE TITLE
Can't play music in iPhone locked state

### ISSUE THREAD

**Issue by qinghou** [tumtumtum/StreamingKit#195]
Hello

I can't play music when iPhone is locked.
After locked and the display of iPhone turn off, then I can't hear music.
When I click home button and iPhone needs to unlock, I resume again.
Hope your kind help.

Regards.

**Comment by duongvanthai** [comment]
I have same issues when STKAudioPlayer play file offline in device. It's stop audio when lock screen over 10s... (STKAudioPlayer state auto change to STKAudioPlayerStateBuffering and stop). But with audio streaming. It's OK. I don't understand. :) How to fix it?

**Comment by hutchy** [comment]
Have you enabled background audio in your app?

A quick Google will show you how to alter the target details or http://www.raywenderlich.com/29948/backgrounding-for-ios

**Comment by duongvanthai** [comment]
Yes, I have enabled background audio in my app. When I streaming file from internet, lock screen playback is OK. This bug only happens when play audio file offline on device. :)

**Comment by WillHu** [comment]
Even when streaming audio from internet, lock screen playback fails. Background audio is also enabled in my app.

**Comment by edl00k** [comment]
What iOS version are you using?

**Comment by WillHu** [comment]
iOS 8.3.1

**Comment by edl00k** [comment]
FYI: also reproduced on iOS 8.1.4, 9.0 beta 2

**Comment by Heisenbean** [comment]
look at [wiki](https://github.com/tumtumtum/StreamingKit/wiki/Background-Audio)

**Comment by GaoHe-1** [comment]
so easy.

**Comment by derpoliuk** [comment]
@qinghou if [wiki page](https://github.com/tumtumtum/StreamingKit/wiki/Background-Audio) helps, please close the issue.
